File: Loja de Games/Model/Cliente.php
<?php
class Cliente 
{
    public $id;
    public $nome;
    public $endereco;
    public $email;
    public $senha;

    public $con;

    public function __construct()
    {
        $conexao = new Conexao();
        $this->con = $conexao->retornarConexao();
    }

    public function Criar()
    {
        $sql = $this->con->prepare("INSERT INTO cliente (id, nome, endereco, email, senha) VALUES (null,?,?,?,?)");
        $sql->execute([$this->nome,$this->endereco,$this->email,$this->senha]);
    }

    public function ListarUmCliente()
    {
        $sql = $this->con->prepare("SELECT * FROM cliente WHERE email = ? AND senha = ?");
        $sql->execute([$this->email,$this->senha]);
        $row = $sql->fetchAll();

        if(sizeof($row) > 0)
        {
            return $row[0];
        }
        else
        {
            return $row;
        }
    }

    public function Login()
    {
        $sql = $this->con->prepare("SELECT * FROM cliente WHERE email = ? AND senha = ?");
        $sql->execute([$this->email,$this->senha]);
        $count = $sql->fetchColumn();

        if($count > 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }

	public function Editar()
    {
        $sql = $this->con->prepare("UPDATE cliente SET nome = ?, endereco = ?, email = ?, senha = ?  WHERE id=?");
		$sql->execute([$this->nome,$this->endereco,$this->email,$this->senha,$this->id]);
	}

	public function Deletar()
    {
		$sql = $this->con->prepare("DELETE FROM cliente WHERE id=?");
		$sql->execute([$this->id]);
	}
}
?>

File: C#/loja_de_games/loja_de_games/Classes/Cliente.cs
namespace loja_de_games.Classes
{
    public static class Cliente
    {
        public static int id { get; set; }
        public static string nome { get; set; }
        public static string email { get; set; }
        public static string endereco { get; set; }
        public static string senha { get; set; }
        public static Conexao con = new Conexao();

        public static void Criar()
        {
            con.ExecutarComando($"INSERT INTO `cliente`(`id`, `nome`, `email`, `endereco`, `senha`) VALUES (null,'{nome}','{email}','{endereco}','{senha}') ");
        }

        public static void Editar()
        {
            con.ExecutarComando($"UPDATE `cliente` SET `nome`= '{nome}',`email`= '{email}',`endereco`= '{endereco}',`senha`= '{senha}' WHERE id = '{id}'");
        }

        public static void Deletar()
        {
            con.ExecutarComando($"DELETE FROM `cliente` WHERE id = {id}");
        }

        public static bool Login()
        {
            var resultado = con.ExecutarConsulta($"SELECT * FROM cliente WHERE email = '{email}' AND senha = '{senha}'");

            if (resultado.Rows.Count > 0)
            {
                id = (int)resultado.Rows[0]["id"];
                nome = (string)resultado.Rows[0]["nome"];
                email = (string)resultado.Rows[0]["email"];
                endereco = (string)resultado.Rows[0]["endereco"];
                senha = (string)resultado.Rows[0]["senha"];

                return true;
            }
            else
            {
                Logoff();
                return false;
            }
        }

        public static void Logoff()
        {
            id = 0;
            nome = "";
            email = "";
            endereco = "";
            senha = "";
        }
    }
}
